File: eBotanika.Model/Biljke.cs
namespace eBotanika.Model
{
    public class Biljke
    {
        public int BiljkeID { get; set; }
        public int KategorijaID { get; set; }
        public Kategorija Kategorija { get; set; }
        public string Naziv { get; set; }
        public string Opis { get; set; }
        public string Slika { get; set; }
        public decimal Cijena { get; set; }
    }
}

File: ebotanika_mobile/lib/models/biljke.dart
class Biljke {
  int biljkeID;
  int kategorijaID;
  String naziv;
  String opis;
  String slika;
  double cijena;

  Biljke(
      {required this.biljkeID,
      required this.kategorijaID,
      required this.naziv,
      required this.opis,
      required this.slika,
      required this.cijena});

  factory Biljke.fromJson(Map<String, dynamic> json) {
    return Biljke(
      biljkeID: int.parse(json["biljkeID"].toString()),
      kategorijaID: json["kategorijaID"],
      naziv: json["naziv"],
      opis: json["opis"],
      slika: json["slika"],
      cijena: json["cijena"],
    );
  }

  Map<String, dynamic> toJson() => {
        "biljkeID": biljkeID,
        "kategorijaID": kategorijaID,
        "naziv": naziv,
        "opis": opis,
        "slika": slika,
        "cijena": cijena
      };
}
